### PR TITLE
[Chore] Replace Cocoapods-keys with Arkana

### PR DESCRIPTION
**Pull request by ducbm051291**
- #70 

## What happened

- We need to replace Cocoapods-keys with Arkana because Cocoapods-keys has some issues somehow it can't retrieve the keys values and it is old
 
## Insight

- Add .arkana.yml file to configurate Arkana
 - Update .env with ClientId and ClientSecret
- Update .gitignore to ignore Arkana generated folder
- Add Arkana in gemfile
- Add Arkana Generated Pods in Podfile

```
Step 1: Run `bundle install` to install Arkana
Step 2: Run `bundle exec arkana` to generate Pods
Step 3: Run `bundle exec pod installl` to install generated Pods
```
 
## Proof Of Work

Screenshot 2022-11-29 at 16 53 51

Screenshot 2022-11-29 at 16 54 23

Screenshot 2022-11-29 at 16 54 06

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,4 @@
-ClientId = ""
-ClientSecret = ""
+ClientIdDebug = "zWRM1gjQrQWu0ohAUyS3pn6Bf5oq4two87B-9JWBc3E"
```
Comment (blyscuit) [inline .env:1]: I think it's better to post these to 1Password. This repo is public so it's really not recommended.
If we add `.env` to gitignore then these values will only need to be filled once 🙏 . You can include the instruction to find the 1Pass in this PR comments or on Slack.

Comment (blyscuit) [reply .env:1]: Also, provide `.env.example` to the repo, so other dev knows what keys to put 😄

**Review by blyscuit — COMMENTED** [review]
Looks like it's on a wrong base branch 🤔

**Comment by ducbm051291** [comment]
> Looks like it's on a wrong base branch 🤔

I'm middle of working login integration task :))

**Pull request closed without merging**